### ISSUE TITLE
Directive Viewer.js is closed abruptly when unrelated element updates

### ISSUE THREAD

**Issue by IlyaSemenov** [mirari/v-viewer#23]
In a code like this:

```vue
<template>
  <div>{{ some }}</div>
  <div v-viewer><img ...></div>
</template>
```

When `some` is updated, Viewer.js added by a directive to a **different** element closes abruptly.

Minimal reproducible case: https://codepen.io/ilya-semenov/pen/qYBjjW

**Comment by IlyaSemenov** [comment]
I realise it probably can't be "fixed" because it's how Vue's `componentUpdated` works. I would appreciate if there at least was a way to disable that hook (e.g. with something like `v-viewer.nowatch`).

I tried to switch to `<viewer>` as it doesn't watch for VDom changes at all, but then I bumped into the need to pass obligatory `<viewer :images="???">` argument (see #24). Unfortunately, I can't even do `<viewer :images="[]">` because this will generate a new array object on every VDom recalculation and fire `watch` in the viewer component. The only workaround for now is `<viewer :images="emptyStubArray">` where emptyStubArray is what is says.
